content/_index.md

```diff
@@ -65,15 +65,8 @@ and create [Bodhi](https://bodhi.fedoraproject.org) updates! [Learn more.](/docs
    and fix bugs as soon as possible, so you can open an [upstream issue](https://github.com/packit/packit-service/issues/new)
    any time.
 
-4. _We are here to help_ - In case of questions, feel free to contact
-   [us](https://github.com/orgs/packit/people):
-
-#### Contact
-
-- **#packit:fedora.im** on [Matrix](https://matrix.org/)
-- **#packit** on [Libera.Chat](https://libera.chat/)
-- **hello@packit.dev**
-- **@packit** on {{< rawhtml >}}<a rel="me" href="https://fosstodon.org/@packit">Fosstodon</a>{{< /rawhtml >}}
+4. _We are here to help_ - In case of questions, feel free to [contact us]({{< ref
+   "faq#how-can-i-contact-you" >}}).
```

Comment:
Sorry. I meant the opposite way. :speak_no_evil: 

(The contact header on the homepage is linked from multiple places in the service so I would preserve that one.)

Reply:
I know :-) I can change the "multiple places" as well.
See my above comment for reasoning.

But I'm not against leaving it the original way if you insist.

Reply:
Reverted to the original layout, the `#contact` is used also in code and presentations.